lib/ansible_spec/load_ansible.rb

```diff
@@ -148,6 +148,8 @@ def self.get_inventory_param(line)
         host['private_key'] = value if key == "ansible_ssh_private_key_file"
         host['user'] = value if key == "ansible_ssh_user"
         host['uri'] = value if key == "ansible_ssh_host"
+        host['pass'] = value if key == "ansible_ssh_pass"
+        host['conn'] = value if key == "ansible_connection"
```

Comment:
1st, I think that name of `conn` is confusing.
Counld you change from `conn` to `connection`?
2nd, I think that initial value of `host['conn']` is necessary. How about you?